### PR TITLE
Add PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by johlju**
- Added a pull request template as PULL_REQUEST_TEMPLATE.md that will be shown
  to the contributor when a pull requests are sent in.

*This is the first part of issue https://github.com/PowerShell/DscResources/issues/427 where all repositories get a pull request template. This repo is special so it gets a template that does not reference resources.*

---
This change is [Reviewable](https://reviewable.io/reviews/powershell/dscresource.tests/285)

### CODE REVIEW

**Comment by johlju** [comment]
@PlagueHO Could I ask you to please to do a review of this?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<!--
+    To aid community reviewers in reviewing and merging your PR, please take
+    the time to run through the below checklist and make sure your PR has
+    everything updated as needed.
+
+    Change to [x] for each task in the task list that applies to your pull
+    request (PR). For those task that don't apply to you pull request (PR),
+    leave those as is.
+-->
+- [ ] Added an entry under the Unreleased section in the CHANGELOG.md. Entry
```
Comment (davetheunissen) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should this be README.md rather than CHANGELOG.md? Looks like the change log is in the README?

**Comment by davetheunissen** [comment]
@johlju no problem, I'll see if I can get some time today to have a look at a couple of the others.

**Comment by PlagueHO** [comment]
Cheers @davetheunissen - will be back at work on Monday! :grin: Thanks @johlju as always!

**Comment by johlju** [comment]
@davetheunissen you are more than welcome! All help is greatly appreciated! 😃

**Pull request merged**